### DISCUSSION TITLE
How to point to the latest .min.js bundle?

### DISCUSSION THREAD

**Discussion by AlphaJack** [squidfunk/mkdocs-material#2773] [Q&A] [answered]
If I create an override of `base.html`, it contains 
```html
 [...]
  <script src="{{ 'assets/javascripts/bundle.3981fc68.min.js' | url }}"></script>
 [...]
```

However, when a new version gets released, the original file is updated and points to another file name:
```html
 [...]
  <script src="{{ 'assets/javascripts/bundle.76f349be.min.js' | url }}"></script>
 [...]
```

The website generated with `mkdocs build` refers to 3981fc68, but only 76f349be is present in the assets directory. This breaks the search and other js features.

Is there a variable to refer to the currently available 'assets/javascripts/bundle.XXXXXX.min.js' file?

**Comment by squidfunk** [comment, marked as answer]
There's currently no variable, the content hashes are generated as part of the build. The general advice is to use template block overrides. Overriding `base.html` may bring a lot of other problems. Thus, you could just extend the respective block with a `super()` call, as we do in the overrides for our own documentation:

https://github.com/squidfunk/mkdocs-material/blob/a4a00af0177e4d50e0af47ae2edb2723b4c50c28/src/overrides/main.html#L73-L79

**Reply by AlphaJack** [reply]
Many thanks!
